x-team/auto-ui
# Upgrade dependencies

**Release Type:** *Dev improvement* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2438

## Description

Run `npm audit` command and followed its suggestions. Also, upgraded critical libraries (like `babel`) to the latest minor versions, and utilities to major ones.

## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Steps to Test or Reproduce

1. Run `npm i` to update your locally installed packages. Do not remove `package-lock`.
2. Check that `npm t` passes, `npm run build` builds the app successfully, `npm start` starts up a playground UI with no errors in the terminal.
3. Finally, integrate into `app` and check if that works. No functionality were affected. Just make sure the app runs.

## Impacted Areas in Application

NPM dependencies only.

## Screenshots

### Before the `audit` and fix applied

![image](https://user-images.githubusercontent.com/579331/47927663-ff275180-ded4-11e8-9b73-0dc47f52b77b.png)

### After the fix was applied and `audit` performed

![image](https://user-images.githubusercontent.com/579331/47928812-55e25a80-ded8-11e8-954c-7a099d4fc27a.png)
